go/common/crypto/address/address.go

```diff
@@ -0,0 +1,80 @@
+// Package address implements a generic cryptographic address.
+package address
+
+import (
+	"bytes"
+	"encoding/base64"
+	"errors"
+
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+)
+
+const (
+	// Addresses are 20 bytes long.
+	Size = 20
+)
+
+var (
+	// ErrMalformed is the error returned when an address is malformed.
+	ErrMalformed = errors.New("hash: malformed address")
```

Comment:
`hash:` isn't the correct prefix to be using for the error message.

Reply:
Yea, fixed.